### PR TITLE
Add ccze

### PR DESCRIPTION
**Pull request by malyn**

### CODE REVIEW

File: pkgs/tools/misc/ccze/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  meta = with stdenv.lib; {
+    description = "Fast, modular log colorizer";
+    longDescription = ''
+      Fast log colorizer written in C, intended to be a drop-in replacement
+      for the Perl colorize tool.  Includes plugins for a variety of log
+      formats (Apache, Postfix, Procmail, etc.).
+    '';
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ malyn ];
+    platforms = platforms.all;
```
Comment (edwtjo) [inline pkgs/tools/misc/ccze/default.nix]: You tested this on all platforms, not just linux?

**Comment by edwtjo** [comment]
Looks good, builds fine.

File: pkgs/tools/misc/ccze/default.nix (hunk 1 above)
Comment (malyn) [inline pkgs/tools/misc/ccze/default.nix]: No, just Linux.  I wasn't sure if this value was supposed to be a list of what was tested or a theory as to where the package might work.  I change it to just "linux" is that is the correct thing to do.

Comment (edwtjo) [inline pkgs/tools/misc/ccze/default.nix]: Please do

Comment (malyn) [inline pkgs/tools/misc/ccze/default.nix]: Done.

**Comment by edwtjo** [comment]
Please squash the commits

**Comment by malyn** [comment]
Done.  BTW, does it help when I ping this PR after pushing a change, or is GitHub already letting you know that something happened here?

**Comment by edwtjo** [comment]
All repo watchers gets notified on new issues, pull requests and comments. Not on modifications. Please purge unrelated commits from the PR.

**Comment by malyn** [comment]
Done.  Sorry for the hassle -- still getting my feet wet.

**Comment by edwtjo** [comment]
No hassle, thank you for the contribution and welcome aboard :)

**Pull request merged**